src/main/java/Messages.java

```diff
@@ -0,0 +1,53 @@
+public class Messages {
+    public static void printHorizontalLine() {
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void printHello() {
+        printHorizontalLine();
+        System.out.println("Hello! I'm Duke. ");
+        System.out.println("What can I do for you?");
+        printHorizontalLine();
+    }
+
+    public static void printGoodBye() {
+        printHorizontalLine();
+        System.out.println("Bye. Hope to see you again soon!");
+        System.out.println("Shutting down...");
+        printHorizontalLine();
+    }
+
+    public static void printAddedToDo(String title) {
+        printHorizontalLine();
+        System.out.println("Done. I've added the following to-do to your list: " + title);
+        printHorizontalLine();
+    }
+
+    public static void printAddedEvent(String title) {
+        Messages.printHorizontalLine();
```

Comment:
Consider removing the Messages. prefix, as it is not needed for a method in the same class

Reply:
Good point. Will do.